scripts/components/Interactions/OpenContent.scss

```
@@ -60,18 +60,19 @@ $labelWidth: 9em;
       &:before {
         font-family: "360-image";
         vertical-align: middle;
-        color: $hoverBg;
+        color: $dragIcon;
+        font-size: 1.25em;
       }
       &--horizontal {
-        left: calc(100% - 0.75em);
-        top: calc(50% - 0.75em);
+        left: calc(100% - 1em);
+        top: calc(50% - 1em);
         &:before {
           content: $arrow-horiz-scale;
         }
       }
       &--vertical {
-        bottom: - 0.75em;
-        left: calc(50% - 0.75em);
+        bottom: - 1em;
+        left: calc(50% - 1em);
```

Using units relative to the element's size will make this more future-proof.

```suggestion
        bottom: 0;
        left: 50%;
        transform: translateX(-50%) translateY(50%);
```